pkg/larker/larker.go

```diff
@@ -112,3 +124,98 @@ func (larker *Larker) Main(ctx context.Context, source string) (string, error) {
 
 	return formattedYaml, nil
 }
+
+func (larker *Larker) Hook(
+	ctx context.Context,
+	source string,
+	name string,
+	arguments []interface{},
+) (*HookResult, error) {
+	if name == "" {
+		return nil, fmt.Errorf("%w: empty hook name specified", ErrSanity)
+	}
+
+	if len(arguments) != 1 {
+		return nil, fmt.Errorf("%w: hook takes exactly 1 argument, got %d", ErrSanity, len(arguments))
+	}
+
+	outputLogsBuffer := &bytes.Buffer{}
+
+	thread := &starlark.Thread{
```

Comment:
I think we need to set some meaningful `SetMaxExecutionSteps` here.

Reply:
Shouldn't https://github.com/cirruslabs/cirrus-cli/pull/303/commits/2711b422c22e4bc2ca140c06f4665b59c2df98c1 cover this already?